client/sites-dashboard/components/sites-site-item-thumbnail.tsx

```diff
@@ -22,10 +23,18 @@ export const SiteItemThumbnail = ( { site, ...props }: SiteItemThumbnailProps )
 
 	const shouldUseScreenshot = status === 'public';
 
+	let siteUrl = site.URL;
+	if ( site.options?.updated_at ) {
+		const updatedAt = new Date( site.options.updated_at );
+		updatedAt.setMinutes( 0 );
+		updatedAt.setSeconds( 0 );
+		siteUrl = addQueryArgs( siteUrl, { v: updatedAt.toISOString() } );
```

Comment:
Unix time would do just as well and is shorter and doesn't need as much escaping (i.e. no colons)

```suggestion
		siteUrl = addQueryArgs( siteUrl, { v: updatedAt.getTime() / 1000 } );
```